Kiosk/ServiceHelpers/InkRecognizer.cs

```diff
@@ -0,0 +1,139 @@
+// 
+// Microsoft Cognitive Services: http://www.microsoft.com/cognitive
+// 
+// Microsoft Cognitive Services Github:
+// https://github.com/Microsoft/Cognitive
+// 
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+// 
+// MIT License:
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED ""AS IS"", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// 
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Data.Json;
+using Windows.Graphics.Display;
+using Windows.UI.Input.Inking;
+
+namespace ServiceHelpers
+{
+    public class InkRecognizer
+    {
+        string inkRecognitionUrl;
+        HttpClient httpClient;
+
+        private IDictionary<uint, InkStroke> StrokeMap { get; set; }
+        private string LanguageCode { get; set; }
+
+        public InkRecognizer(string subscriptionKey, string endpoint, string inkRecognitionUrl)
+        {
+            httpClient = new HttpClient() { BaseAddress = new Uri(endpoint) };
+            httpClient.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+
+            this.inkRecognitionUrl = inkRecognitionUrl;
+
+            StrokeMap = new Dictionary<uint, InkStroke>();
+            LanguageCode = "en-US";
+        }
+
+        public void AddStrokes(IReadOnlyList<InkStroke> strokes)
+        {
+            foreach (var stroke in strokes)
+            {
+                StrokeMap[stroke.Id] = stroke;
+            }
+        }
+
+        public void ClearStrokes()
+        {
+            StrokeMap.Clear();
+        }
+
+        public void SetLanguage(string languageCode)
+        {
+            LanguageCode = languageCode;
+        }
+
+        public JsonObject ConvertInkToJson()
+        {
+            // If needed to use the device's DPI, and example is below. Whatever DPI is used below will need to be used again when handling the response's ink points.
```

Comment:
>// If needed to use the device's DPI [](start = 12, length = 36)

why would you *not* use the device's DPI?  I'd be confused here if I was reading this comment and was new to this area...